Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -346,6 +351,26 @@ - (BOOL)handleDeepLink:(NSURL *)url {
     return handled;
 }
 
+- (BOOL)continueUserActivity:(NSUserActivity *)userActivity {
+    if ([userActivity.activityType isEqualToString:NSUserActivityTypeBrowsingWeb]) {
+        self.preferenceHelper.universalLinkUrl = [userActivity.webpageURL absoluteString];
```

Comment:
This won't work for custom subdomains (which I think you're already probably aware of). I think the backend may always generate these urls with just bnc.lt, though, so it might work.